UndertaleModLib/Decompiler/Disassembler.cs

```diff
@@ -34,18 +35,24 @@ public static string GenerateLocalVarDefinitions(this UndertaleCode code, IList<
 
             return sb.ToString();
         }
-
+        
```

Comment:
And this one

Reply:
Also removed, no clue how that got in there.